### PR TITLE
Update Forum Helpers Page JavaScript

### LINKED ISSUE #65: Feature Request: ocular status
hello it's me business man jeffalo, i am here to suggest showing ocular status under bios on the user list.

**Is your feature request related to a problem?**
i like ocular status

**Describe alternatives you've considered**
n/a ocular is too original

**Additional context**
https://my-ocular.jeffalo.net/docs

### PR DESCRIPTION
**Pull request by Accio1**
### Resolves:

This resolves #65 and most of #31

### Changes:
-Switch to fetch() instead of XMLRequests
-Add ocular statuses to the list of Forum Helpers page.
-Add the ability to get post counts from ScratchDB, but that seems to be down right now, so it is currently disabled.
-Display the number of curators, managers, and total members on the list of Forum Helpers page.

### Local Tests:

Tested Locally

@jeffalo @leahcimto

### CODE REVIEW

**Comment by Accio1** [comment]
Bad bot

File: forumhelpers/FHP.js [hunk 1]
```diff
@@ -1,53 +1,60 @@
...
+				var bio = data[j].bio;
+				var section = document.getElementById(dataSet+"List").innerHTML;
+				var addition = `
+				<div class="profileContainer" id="${name}">
+					<h4 class="profileName"><a href="https://scratch.mit.edu/users/${name}/">${name}</a><span id="${name}Posts"></span></h4>
+					<div class="profileInner">
+						<a href="https://scratch.mit.edu/users/${name}/"><img src="https://cdn2.scratch.mit.edu/get_image/user/${id}_60x60.png" class="profilePicture" id="${name}pfp" loading="lazy" alt="${name}'s Profile Picture"></a>
+						<p class="profileBio">${bio}</p>
+					</div>
+					<p class="FullOcular" id="${name}Ocular" style="border: 5px solid black; border-radius: 10px; padding: 5px; margin-right:20%; width: fit-content" title="Ocular Status"><span class="ocularStatus" id="${name}Status">Loading Status...</span></p>
```
Comment (leahcimto) [inline forumhelpers/FHP.js:19]: Could you use External CSS instead of inline?

Comment (Accio1) [reply forumhelpers/FHP.js:19]: Right, I was so intent on getting the CSS for that to look right that I forgot to move it to the stylesheet.

**Review by leahcimto — APPROVED** [review]
Just previewed it locally and it looks amazing!

**Comment by Accio1** [comment]
> Just previewed it locally and it looks amazing!

Thanks! I'm going to wait for @jeffalo just to make sure that he also agrees since it is his site that the data is being pulled from.

**Comment by jeffalo** [comment]
just checked it out, looks cool.

honestly, you don't need the delay between fetching my-ocular. i can take care of things on my end to make sure im not getting unintentionally (d)dos'd.

**Review by FunctionalMetatable — COMMENTED** [review]
I think you can use /v3/forum/user/info instead of /v2/forum/user/info now. (wait am I allowed to review this)

**Comment by Accio1** [comment]
@jeffalo I've removed the delay for fetching Ocular Statuses.
@FunctionalMetatable thanks! I've changed the link and it is working now! I added a half second delay for fetching post counts from ScratchDB so that it doesn't spam their servers.

Since this has been approved, I'm going to go ahead and merge. This page still requires a few small updates, namely making the post count more distinct, but those can be worked on in the future.

**Comment by leahcimto** [comment]
Yay! 🎉

**Pull request merged**